schema/opentelemetry_configuration.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$id": "https://opentelemetry.io/otelconfig/OpenTelemetryConfiguration",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "OpenTelemetryConfiguration",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "file_format": {
+            "type": "number"
```

Comment:
Oops, was doing a last pass over and noticed this. Should this not be a string?

Reply:
`file_format` refers to the version, e.g. `0.1`.

Reply:
Right, I mean do we not want a string instead of a float for a version number? Even if only using 2 digits instead of 3 I'd expect a string.